recipes/paraprobe/LICENSE

```diff
@@ -0,0 +1,32 @@
+BSD 3-Clause License
```

Comment:
The license in the `summary` is `GPL` but the license you uploaded is `BSD`, which one is the correct one?

Reply:
I changed it to be consistent with BSD. But we need confirmation from the author. I have raised an issue: https://gitlab.com/paraprobe/paraprobe-toolbox/-/issues/32

Reply:
@mkuehbach Can you clarify this?

Reply:
Clarified to be GPL-v3; updated accordingly